readme.md

```diff
@@ -160,9 +157,9 @@ describe('Products Service', function() {
 
 ### :clap: Doing It Right Example: A test structured with the AAA pattern
 
-![](https://img.shields.io/badge/🔧%20Example%20using%20Jest-blue.svg
- "Examples with Jest") ![](https://img.shields.io/badge/🔧%20Example%20using%20Mocha-blue.svg
- "Examples with Jest")
+![Jest example](https://img.shields.io/badge/🔧%20Example%20using%20Jest-blue.svg
+ "Examples with Jest") ![Mocha example](https://img.shields.io/badge/🔧%20Example%20using%20Mocha-blue.svg
+ "Examples with Moch")
```

Comment:
Should be *Mocha* instead of *Moch*? :)

Reply:
Yup, thanks! Noticed a couple more while I was fixing too 😄